src/controllers/AuthController.js

```diff
@@ -31,6 +36,57 @@ const forgotPassword = async (request, response) => {
   }
 };
 
-export default {
-  forgotPassword
+/**
+ * Update user verified status
+ *
+ * @param {object} req
+ * @param {object} res
+ * @returns {json} - json
+ */
+
+const updateStatus = async (req, res) => {
+  const { params: { token } } = req;
+
+  const tokenValue = await verifyToken(token);
```

Comment:
There's an existing verifyToken middleware that you could use to verify the token before it gets to the controller instead of creating a similar helper and using it here. Using that middleware could save you a lot of code from line 50 to 64.

Reply:
fixed